lib/transforms/reviewSubResourceIntegrity.js

```diff
@@ -0,0 +1,53 @@
+var crypto = require('crypto');
+
+module.exports = function (queryObj, options) {
+    options = options || {};
+    return function reviewSubResourceIntegrity(assetGraph) {
+        assetGraph.findAssets(queryObj || { type: 'Html', isInline: false, isFragment: false, isLoaded: true }).forEach(function (htmlAsset) {
+            var changed = false;
+            assetGraph.findRelations({from: htmlAsset, type: ['HtmlStyle', 'HtmlScript'], to: {isInline: false, isLoaded: true}}).forEach(function (relation) {
```

Comment:
Only check the loaded ones? There might be a use case for making sure a CDN included asset has the correct identity. Of course this is mostly up the the implementor based on what relationquery is used to populate the graph. Our default queries used in most assetgraph tools usually don't populate external assets, so a security check would be potentially lost here